src/mmw/js/src/analyze/models.js

```diff
@@ -472,15 +492,48 @@ function createAnalyzeTaskGroupCollection(aoi, wkaoi) {
             .value();
     }
 
+    // Remove PA only tasks first, if needed,
+    // as it is always a lazy-load option.
+    if (!utils.isInPAStrict(aoi)) {
+        var isPATask = function(task) {
+            return task.name.startsWith('pa_');
+        };
+
+        taskGroups = _(taskGroups)
+            // Remove tasks that are PA only
+            .map(function(tg) {
+                tg.tasks = _.reject(tg.tasks, isPATask);
+                return tg;
+            })
+            // Remove task groups with no tasks
+            .filter(function(tg) {
+                return !_.isEmpty(tg.tasks);
+            })
+            .value();
+    }
+
     if (!utils.isInDrb(aoi)) {
         var isDrbTask = function(task) {
             return task.name.startsWith('drb');
         };
+         var isPointSource = function(task) {
+            return task.name.endsWith('pointsource');
+        };
 
         taskGroups = _(taskGroups)
             // Remove tasks that are DRB only
             .map(function(tg) {
                 tg.tasks = _.reject(tg.tasks, isDrbTask);
+
+                // DRB is the point source analysis default task.
+                // Mark new first task in point source task group
+                // as not lazy so the UI has something to show.
+                if(isPointSource){
+                    if (tg.tasks.length > 0) {
+                        tg.tasks[0].lazy = false;
+                    }
+                }
+
```

Comment:
Instead of greying out invalid options in an analysis selector dropdown the existing setup is to remove options from the dropdown entirely. I decided to continue with that convention for consistency.